File: Common/InMotionGIT.Common/Helpers/MD5Helper.cs
using System;
using System.IO;
using System.Security.Cryptography;

namespace InMotionGIT.Common.Helpers
{

    public class MD5Helper
    {

        public static string CheckSum(string path)
        {
            string result = "";
            try
            {
                var manager = MD5.Create();
                var content = File.OpenRead(path);
                result = BitConverter.ToString(manager.ComputeHash(content)).Replace("-", string.Empty);
                content.Close();
            }
            catch (Exception ex)
            {

            }

            return result;
        }

    }

}

File: Common/InMotionGIT.Common - Copy/Helpers/MD5Helper.vb
Imports System.IO
Imports System.Security.Cryptography

Namespace Helpers

    Public Class MD5Helper

        Public Shared Function CheckSum(path As String) As String
            Dim result As String = ""
            Try
                Dim manager = MD5.Create()
                Dim content = File.OpenRead(path)
                result = BitConverter.ToString(manager.ComputeHash(content)).Replace("-", String.Empty)
                content.Close()
            Catch ex As Exception

            End Try

            Return result
        End Function

    End Class

End Namespace